data/mock-27/README.md

```diff
@@ -0,0 +1,23 @@
+# mock-nifH-Even
+This mock community was comprised of DNA from the following six diazotrophic species: Anabaena torulosa (Carm.) Lagerh. (Cyanobacteria) Algae Collection Vienna (ASW 01028), Desulfosporosinus acidophilus strain SJ4T (DSM22704) (Firmicutes) (Alazard et al., 2010), Kosakonia sacchari (Gammaproteobacteria) (in house strain, 99.89% sequence identical (1105 bp) to K. sacchari SP1T (Chen et al., 2014)), Mesorhizobium loti strain R7A (Alphaproteobacteria) (Kelly et al., 2014), Nostoc microscopicum Vaucher (Cyanobacteria) Algae Collection Vienna (ASW 01020), and Telmatospirillum siberiense 26-4b1T (Alphaproteobacteria) (Hausmann et al., in press). DNA was extracted using Qiagen DNeasy Blood and Tissue Kit (Qiagen) according to the manufacturer’s instructions except that 200 µl of culture was first transferred to Lysing Matrix A tube and cells were disrupted using bead beating (30 s at 4 m s-1; FastPrep-24, MP Biomedicals) after adding PBS and AL buffers. DNA was then used for PCR amplification of the nifH gene using primers Ueda19-R6 as described above. PCR products were cloned using TOPO TA Cloning Kit (Thermo Fisher Scientific). One clone containing nifH gene fragment from each strain was used for colony PCR amplification using primers M13, flanking the insert region.
+Following amplification and purification with ZR-96 DNA Clean-up Kit™ (Zymo) PCR products were quantified using Quant-iT™ PicoGreen® dsDNA Assay Kit (Thermo Fisher Scientific) on a Tecan Safire plate reader (Tecan) and pooled in equal and also tiered proportions.
+
```

Comment:
Please add an additional paragraph detailing that the taxonomy of all mock community members was validated by Sanger, and these IDs appear in the "source" taxonomy. Please also describe what is contained in the CART file, and what the SILVA taxonomies contain (please add a note that this is not a 16S dataset and the SILVA database will not be useful for classification of these sequences).

Here and in the mock-28 readme